www/docs/new/debates.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Presumably not needed :) (And I think some other files can be removed, e.g. hansard_calendar?)